strings.txt

```diff
@@ -16217,10 +16217,12 @@
 
   [editor_storey_number]
     tags = ios, android
+    en = Storey number (maximum %1$@)
     ru = Количество этажей (максимум %1$@)
 
   [error_enter_correct_storey_number]
     tags = ios, android
+    en = Edit buildings with maximum of %1$@ floors
     ru = Редактируйте здания высотой максимум %1$@ этажей.
```

Comment:
Ну, можно и короче записать, ок.

> On Apr 21, 2016, at 4:01 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/2947#discussion_r60574842>:
> 
> >      ru = Количество этажей (максимум %1$@)
> >  
> >    [error_enter_correct_storey_number]
> >      tags = ios, android
> > +    en = Edit buildings with maximum of %1$@ floors
> >      ru = Редактируйте здания высотой максимум %1$@ этажей.
> %1 нужен если подставляемых параметров больше одного.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2947/files/44b3f477a158544a38212dbfd95100ddbb3c305c#r60574842>